.idea/transbank-sdk-ruby-onepay-example.iml

```diff
@@ -23,14 +23,78 @@
       <excludeFolder url="file://$MODULE_DIR$/vendor/bundle" />
```

Comment:
Are the files from the IDE supposed to be part of the git repo? 

I don't think so: They will get out of sync quite easily (unless everybody uses the same IDE, which we aren't)